### PR TITLE
Redis empty request

### PR DESCRIPTION
**Pull request by lzfhust**
issue: https://github.com/apache/incubator-brpc/issues/1743
in brpc redis protocol, set failed and return error message when byte size is empty in redis request.

### CODE REVIEW

File: src/brpc/policy/redis_protocol.cpp [hunk 1]
```diff
@@ -305,6 +305,11 @@ void SerializeRedisRequest(butil::IOBuf* buf,
         return cntl->SetFailed(EREQUEST, "The request is not a RedisRequest");
     }
     const RedisRequest* rr = (const RedisRequest*)request;
+    // If redis byte size is zero, brpc call will fail with E22. Continuous E22 may cause E112 in the end.
+    // So set failed and return useful error message
+    if(rr->ByteSize() == 0) {
```
Comment (wwbmmm) [inline src/brpc/policy/redis_protocol.cpp:310]: if 后面加个空格

Comment (lzfhust) [reply src/brpc/policy/redis_protocol.cpp]: > if 后面加个空格

done

**Comment by wwbmmm** [comment]
LGTM

**Comment by serverglen** [comment]
LGTM

**Pull request merged**